package.json

```diff
@@ -21,6 +21,7 @@
     "build-admin-panel": "yarn workspace @tupaia/admin-panel build",
     "build-internal-dependencies": "./scripts/bash/buildInternalDependencies.sh",
     "build-web-frontend": "yarn workspace @tupaia/web-frontend build",
+    "download-env-vars": "./scripts/bash/downloadEnvironmentVariables.sh",
```

Comment:
These now come from LastPass, which will make things _so_ much easier to manage!